ndmg/stats/qa_reg.py

```diff
@@ -152,33 +156,61 @@ def plot_brain(brain, minthr=2, maxthr=95, edge=False):
 
 
 def plot_overlays(atlas, b0, cmaps=None, minthr=2, maxthr=95, edge=False):
+    """
+    A function to plot the overlay figures of registered and reference brain slices.
+    
+    Parameter
+    ---------
+    atlas: str, nifti image, numpy.ndarray
+        an object to open the data for a registered brain. Can be a string (path to a brain file),
+        nibabel.nifti1.nifti1image, or a numpy.ndarray.
+
+    b0: str, nifti image, numpy.ndarray
+        an object to open the data for a reference brain. Can be a string (path to a brain file),
+        nibabel.nifti1.nifti1image, or a numpy.ndarray.
+
+    cmap: Colormap objects based on lookup tables using linear segments.
+    minthr: int
+    maxthr: int
+    edge: bool
```

Comment:
You need to explain what these inputs are

Reply:
see https://github.com/neurodata/ndmg/pull/372/commits/3be6eafbddb55f52d9bb5ed4b5df981b975ac532